docs/griptape-framework/drivers/image-query-drivers.md

```diff
@@ -91,15 +91,15 @@ engine.run("Describe the weather in the image", [image_artifact])
 !!! info
     In order to use the `gpt-4-vision-preview` model on Azure OpenAI, the `gpt-4` model must be deployed with the version set to `vision-preview`. More information can be found in the [Azure OpenAI documentation](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/gpt-with-vision).
 
-The [AzureOpenAiVisionImageQueryDriver](../../reference/griptape/drivers/image_query/azure_openai_vision_image_query_driver.md) is used to query images using the Azure OpenAI Vision API. Here is an example of how to use it:
+The [AzureOpenAiVisionImageQueryDriver](../../reference/griptape/drivers/image_query/azure_openai_image_query_driver.md) is used to query images using the Azure OpenAI Vision API. Here is an example of how to use it:
 
 ```python
 import os
-from griptape.drivers import AzureOpenAiVisionImageQueryDriver
+from griptape.drivers import AzureOpenAiImageQueryDriver
 from griptape.engines import ImageQueryEngine
 from griptape.loaders import ImageLoader
 
-driver = AzureOpenAiVisionImageQueryDriver(
+driver = AzureOpenAiImageQueryDriver(
```

Comment:
`AZURE_OPENAI_ENDPOINT_3` and `AZURE_OPENAI_API_KEY_3` can be changed to use `_2`, and the model and azure_deployment can be changed to `gpt-4o`. sorry should have clarified this.